src/t_list.c

```diff
@@ -801,6 +851,56 @@ void lremCommand(client *c) {
     addReplyLongLong(c,removed);
 }
 
+/* LREM <key> <count> <index> */
+void lrematCommand(client *c) {
```

Comment:
I wonder if `listTypeDelRange()` and `listElementsRemoved()` functions can be used to implement this and avoid code duplication?